FinalProject.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head>
+     <meta charset="utf-8">
+     <title>Final Project</title>
+ </head>
+ <body>
+     <header>
+     <h1>Faiza Siddiqi</h1>
+     <nav>
+        <a href="http://www.firefox.com">One</a>
+        <a href="http://www.facebook.com">Two</a>
+        <a href="http://www.google.com">Three</a>
+        <a href="http://www.github.com/faizasidd">Github</a>
+        <br>
+</nav>
+     </header>
+     <section>
+        <h2>Favorite Foods</h2>
+        <ul>
+            <li>Pad Thai</li>
+            <li>Chicken Wings</li>
+            <li>Lychee Bubble Tea</li>
+            <li>Strawberry Cheesecake</li>
+        </ul>
+    </section>
+    <section>
+        <h2>Achievements</h2>
+        <p>Progress in this course (100%)<progress value="100" max="100"></progress><br>
+        Progress in the Specialization capstone (20%)<progress value="20" max="100"></progress><br>
+        Progress in life goals (30%)<progress value="30" max="100"></progress></p>
+    </section>
+    <section>
+        <h2>More About Me</h2>
+        <details>
+            <summary>My Childhood</summary>Im from Toronto Canada. I grew up in a big family and had a cat name Simba. 
```

Comment:
Nitpick: Grammar syntax
```suggestion
            <summary>My Childhood</summary>I'm from Toronto Canada. I grew up in a big family and had a cat name Simba. 
```

Reply:
Should the description of your childhood be in `<p>` tag?